### PR TITLE
Add vwop_vv vwop_vx vwop_wx vwop_vv series instruction

### PR DESCRIPTION
**Pull request by joii2020**

### CODE REVIEW

File: cases/src/misc.rs [hunk 1]
```diff
@@ -135,6 +135,12 @@ pub fn shrink_to_imm(x: u64) -> i8 {
     }
 }
 
+// 0 to 15
+pub fn shrink_to_imm_u(x: u64) -> u8 {
```
Comment (XuJiandong) [inline cases/src/misc.rs]: The `imm` has 5 bits, but 16 only takes 4.

Comment (XuJiandong) [inline cases/src/misc.rs]: imm has 5 bits. but 16 only takes 4.

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: cases/src/misc.rs (hunk 1 above)
Comment (joii2020) [reply cases/src/misc.rs]: This function is only used for immediate instruction's testcase. Each intruction needs to be manually coded, so here I only use 0~15.

Comment (XuJiandong) [reply cases/src/misc.rs]: I mean, all cases should be covered.

Comment (joii2020) [reply cases/src/misc.rs]: I'm sorry, I misunderstood this a bit before, I'm working on it.

**Review by XuJiandong — APPROVED** [review]

**Pull request merged**